coverlib/profiles.go

```diff
@@ -68,3 +71,15 @@ func (p *Profiles) String() string {
 	}
 	return buf.String()
 }
+
+// MergeWith merges in the given profiles.
+//
+// For any given line, the resulting hit count is the maximum between the hit
```

Comment:
Comment is stale now that `MergeWith` computes the sum of the hit counts.

Reply:
Good catch, fixed. TFTR!